### ISSUE TITLE
training stuck at some step with Nuscenes train dataset

### ISSUE THREAD

**Issue by fyaft2012** [traveller59/second.pytorch#389]
Hi @traveller59 , when I try to train network by using Nuscenes dataset, it works properly. However, it stuck at step =2300, and show no any information.
I train with "script_server.py" at comment line, this problem happens for both "all.pp.lowa.config" and "all.pp.mida.config". 
 
**python script_server.py**
-------------------------------------------------------------------------------------------------------------------------
num parameters: 66
Load pretrained parameters:
global_step torch.Size([1])
voxel_feature_extractor.pfn_layers.0.linear.weight torch.Size([64, 9])
voxel_feature_extractor.pfn_layers.0.norm.weight torch.Size([64])
voxel_feature_extractor.pfn_layers.0.norm.bias torch.Size([64])
voxel_feature_extractor.pfn_layers.0.norm.running_mean torch.Size([64])
voxel_feature_extractor.pfn_layers.0.norm.running_var torch.Size([64])
voxel_feature_extractor.pfn_layers.0.norm.num_batches_tracked torch.Size([])
rpn.blocks.0.1.weight torch.Size([64, 64, 3, 3])
rpn.blocks.0.2.weight torch.Size([64])
rpn.blocks.0.2.bias torch.Size([64])
...........
...........
train_input_reader {
    batch_size: 3
    dataset {
      kitti_info_path: "/media/lion/FANGYANG/NUSCENES/data/trainval/infos_train.pkl"
      kitti_root_path: "/media/lion/FANGYANG/NUSCENES/data/trainval"
      dataset_class_name: "NuScenesDatasetD8"
    }
    preprocess {
      max_number_of_voxels: 25000
      groundtruth_localization_noise_std: 0.0
      groundtruth_localization_noise_std: 0.0
      groundtruth_localization_noise_std: 0.0
      groundtruth_rotation_uniform_noise: 0.0
      groundtruth_rotation_uniform_noise: 0.0
      global_rotation_uniform_noise: 0.0
      global_rotation_uniform_noise: 0.0
      global_scaling_uniform_noise: 0.95
      global_scaling_uniform_noise: 1.05
      global_translate_noise_std: 0.2
      global_translate_noise_std: 0.2
      global_translate_noise_std: 0.2
      num_workers: 3
      anchor_area_threshold: -1.0
      remove_points_after_sample: true
      groundtruth_drop_max_keep_points: 15
      global_random_rotation_range_per_object: 0.0
      global_random_rotation_range_per_object: 0.0
      database_sampler {
        database_info_path: "/media/lion/FANGYANG/NUSCENES/data/trainval/kitti_dbinfos_train.pkl"
        sample_groups {
          name_to_max_num {
            key: "bus"
            value: 4
          }
        }
        sample_groups {
          name_to_max_num {
            key: "trailer"
            value: 4
          }
        }
        sample_groups {
          name_to_max_num {
            key: "truck"
            value: 4
          }
        }
        global_random_rotation_range_per_object: 0.0
        global_random_rotation_range_per_object: 0.0
        rate: 1.0
      }
      random_flip_x: true
      random_flip_y: true
      sample_importance: 1.0
    }
  }
  train_config {
    optimizer {
      adam_optimizer {
        learning_rate {
          one_cycle {
            lr_max: 0.003
            moms: 0.95
            moms: 0.85
            div_factor: 10.0
            pct_start: 0.4
          }
        }
        weight_decay: 0.01
      }
      fixed_weight_decay: true
    }
    steps: 58650
    steps_per_eval: 5865
    save_checkpoints_secs: 1800
    save_summary_steps: 10
    loss_scale_factor: -1.0
    clear_metrics_every_epoch: true
  }
  eval_input_reader {
    batch_size: 1
    dataset {
      kitti_info_path: "/media/lion/FANGYANG/NUSCENES/data/trainval/infos_val.pkl"
      kitti_root_path: "/media/lion/FANGYANG/NUSCENES/data/trainval"
      dataset_class_name: "NuScenesDataset"
    }
    preprocess {
      max_number_of_voxels: 30000
      num_workers: 2
      anchor_area_threshold: -1.0
    }
  }

WORKER 0 seed: 1594270991
WORKER 1 seed: 1594270992
WORKER 2 seed: 1594270993
runtime.step=50, runtime.steptime=2.551, runtime.voxel_gene_time=0.01594, runtime.prep_time=0.4776, loss.cls_loss=787.1, loss.cls_loss_rt=127.2, loss.loc_loss=2.779, loss.loc_loss_rt=2.593, loss.loc_elem=[0.12, 0.119, 0.4233, 0.05143, 0.06731, 0.07344, 0.4419], loss.cls_pos_rt=114.1, loss.cls_neg_rt=13.11, loss.dir_rt=0.6997, rpn_acc=0.7106, pr.prec@10=0.001186, pr.rec@10=1.0, pr.prec@30=0.001196, pr.rec@30=0.9975, pr.prec@50=0.001193, pr.rec@50=0.2901, pr.prec@70=0.0, pr.rec@70=0.0, pr.prec@80=0.0, pr.rec@80=0.0, pr.prec@90=0.0, pr.rec@90=0.0, pr.prec@95=0.0, pr.rec@95=0.0, misc.num_vox=55176, misc.num_pos=67, misc.num_neg=49899, misc.num_anchors=50000, misc.lr=0.0003, misc.mem_usage=61.4
.........
.........
runtime.step=2300, runtime.steptime=2.506, runtime.voxel_gene_time=0.009616, runtime.prep_time=0.521, loss.cls_loss=0.5468, loss.cls_loss_rt=0.4535, loss.loc_loss=1.887, loss.loc_loss_rt=1.421, loss.loc_elem=[0.04765, 0.08436, 0.161, 0.04663, 0.08473, 0.05603, 0.23], loss.cls_pos_rt=0.238, loss.cls_neg_rt=0.2155, loss.dir_rt=0.7028, rpn_acc=0.9988, pr.prec@10=0.03475, pr.rec@10=0.5823, pr.prec@30=0.4879, pr.rec@30=0.04238, pr.prec@50=1.0, pr.rec@50=9.518e-06, pr.prec@70=0.0, pr.rec@70=0.0, pr.prec@80=0.0, pr.rec@80=0.0, pr.prec@90=0.0, pr.rec@90=0.0, pr.prec@95=0.0, pr.rec@95=0.0, misc.num_vox=71590, misc.num_pos=82, misc.num_neg=49871, misc.num_anchors=50000, misc.lr=0.0003635, misc.mem_usage=62.0
WORKER 0 seed: 1594277004
WORKER 1 seed: 1594277005
WORKER 2 seed: 1594277006
--------------------------------------------------------------------------------------------------------------------------
It just stuck at this point, and never change.

Could you help me with it? Thank you very much! @traveller59
